_private/CreateSession.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE hml>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Create Session</title>
+	<link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
+	<link href="content/style.css" rel="stylesheet">
+</head>
+
+<body>
+	<nav class="navbar navbar-inverse">
+		<div class="navbar-brand">Buzzer</div>
+	</nav>
+	<div class="container">
+		<div class="row">
+			<div class="col-xs-10 col-xs-offset-1 col-sm-9 col-sm-offset-2">
+				<h2>Create New Session</h2>
+				<form class="form">
+					<div class="form-group row">
+						<label class="col-sm-3 col-form-label" for="session_name">Session Name</label>
+						<div class="col-sm-9">
+							<input class="form-control" type="text" id="session_name" name="session_name" />
+						</div>
+					</div>
+					<div class="form-group row">
+						<div class="col-xs-4 col-xs-offset-2 pull-right">
+							<input onclick="showTeamOptions()" type="checkbox" value="1" name="Teams" />
```

Comment:
It's generally bad practise to wire event handlers like this. The main reason is that it trumps and causes issue with handlers added dynamically. In addition, it's mixing code and markup which decreases the maintainability. In my experience it's always better to have a view/(controller/presenter), so a separate js file just for this UI and usually another for for all UIs (site/global etc). By doing this it means logic can be refactored into global and reused easily with the bonus of increased maintainability.